src/extension.ts

```diff
@@ -107,7 +107,7 @@ function getPathPickerMenuItemShortcuts(workspaceFolders: readonly WorkspaceFold
     const activeTextEditorPath = path.dirname(window.activeTextEditor.document.uri.fsPath)
 
     if (activeWorkspaceFolder && activeWorkspaceFolder !== activeTextEditorPath) {
-      shortcuts.push({
+      shortcuts.unshift({
```

Comment:
I cannot suggest an edit but I think it would be easier to read to avoid the `unshift` and instead start from an empty `PathPickerMenuItem` array, then if we have an active editor push the active folder, and then push the workspaces? Any thoughts?

Reply:
![image](https://github.com/HiDeoo/create/assets/44113587/62d3cb03-9ed1-4cc2-81d1-4dbc13e56877)
What do you think?

Reply:
Ah yeah, this is exactly what I had in mind 👍 

> And why did you compare active editor path with active workspace root?

Imagine you have a workspace opened and your active file is at the root of the workspace. Without the comparison, an entry for `/` would appear twice (one for the workspace root and one for the active editor).

Reply:
Updated!

Reply:
I suggest that we eliminate the comparison and filter out duplications. For example, when I press Ctrl+Alt+O+Enter, my expectation is that the current folder is selected. This experience should be consistent.

Reply:
Ah, I think I see what you mean, altho I rarely use multiple workspaces and I think this would only apply in this case but I think what you're suggesting makes sense.

If I get that correctly, we could I think indeed remove this condition and instead use a similar one to filter out the `workspaceFolders` mapped array below to remove out the identical workspace root.

Reply:
![image](https://github.com/HiDeoo/create/assets/44113587/bc2bcf4c-d6b3-434f-839d-4419317f4afe)
@HiDeoo How's this?

Reply:
I _think_ this is indeed the correct condition we would want altho I always have a hard time reasoning with code screenshots ^^

I think you can push this change and no matter what, I'll play with it locally anyway to get a better idea 👍

Reply:
Pushed 🙌. Just to make sure I won't pollute the commit history.